### DISCUSSION TITLE
Tag Cloud with Varying Sized Words Based on Total References

### DISCUSSION THREAD

**Discussion by thw26** [zim-desktop-wiki/zim-desktop-wiki#2051] [Ideas]
I have a request for a new option for the Tags plugin.

Similar to this request: #515, I would like to see the Tags side pane either as the default or through a selectable option through Edit > Preferences > Plugins option render the most frequently used Tags with a variable size. For example, a tag with 100 references would have a bigger text size than a tag with 50 references, which would be bigger than a tag with 20 references, and so on. This would be similar to RedNotebook's implementation of a tag cloud.

**Comment by Nick222** [comment]
Good idea !!!

**Comment by Nick222** [comment]
RedNotebook https://rednotebook.sourceforge.io/

**Comment by Nick222** [comment]
See sample https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/515
